locations/spiders/petsathome_gb.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+import scrapy
+import json
+import csv
+from locations.linked_data_parser import LinkedDataParser
+from scrapy.http import FormRequest
+
+
+# This method could be moved into library code and documented, tested etc, supports calls of the form:
+# get_locations("eu_centroids_40km_radius_country.csv")
+# get_locations("eu_centroids_40km_radius_country.csv", ["GB", "IE"])
+# get_locations("us_centroids_50mile_radius_state.csv", "NY")
+def get_locations(areas_csv_file, area_field_filter=None):
+    def get_key(row, keys):
+        for key in keys:
+            if row.get(key):
+                return row[key]
+        return None
+
+    if area_field_filter and type(area_field_filter) is not list:
+        area_field_filter = [area_field_filter]
+    with open("./locations/searchable_points/{}".format(areas_csv_file)) as points:
+        for row in csv.DictReader(points):
+            lat = get_key(row, ["latitude", "lat"])
```

Comment:
I think `get_key` here is an over complication. I like the idea in principle, but here we control the source data. We should just unify on `latitude` or `lat`.

`head -n1 locations/searchable_points/*` shows them all using `latitude,longitude` except for `germany_centroids_80km_radius_country.csv` and `germany_grid_15km.csv`. Lets fix that, and make sure to update any spiders that use them.

Reply:
I would propose that we have this discussion and changes in a subsequent PR that pulled this method out. I don't want to do too much in a single change.

Reply:
Will be fixed with #3796